### PR TITLE
docs: fix readme urls

### PR DESCRIPTION
**Pull request by erights**
See https://github.com/endojs/endo/pull/1451#pullrequestreview-1263981440

@kriskowal does this reflect a problem in repackage.sh ?

### CODE REVIEW

File: packages/exo/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "keywords": [],
   "author": "Endo contributors",
   "license": "Apache-2.0",
-  "homepage": "https://github.com/endojs/endo/tree/master/packages/exo#readme",
+  "homepage": "https://github.com/endojs/endo/blob/master/packages/exo/README.md",
```
Comment (gibson042) [inline packages/exo/package.json:8]: Is the [exo README](https://github.com/endojs/endo/blob/master/packages/exo/README.md) intentionally empty?

File: packages/lp32/package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
   ],
   "author": "Endo contributors",
   "license": "Apache-2.0",
-  "homepage": "https://github.com/endojs/endo/tree/master/packages/netstring#readme",
+  "homepage": "https://github.com/endojs/endo/blob/master/packages/netstring/README.md",
```
Comment (gibson042) [inline packages/lp32/package.json:19]:
```suggestion
  "homepage": "https://github.com/endojs/endo/blob/master/packages/lp32/README.md",
```

**Review by gibson042 — APPROVED** [review]
Looks good after fixing lp32/package.json.

File: packages/exo/package.json (hunk 1 above)
Comment (erights) [reply packages/exo/package.json:8]: Was not intentional, just not yet written. So I added the text for my recent empty ones: pass-style, patterns, exo. Minimal but much better than nothing. It's a start. Done.

File: packages/lp32/package.json (hunk 2 above)
Comment (erights) [reply packages/lp32/package.json]: Done

**Comment by kriskowal** [comment]
Yes, to use this style going forward, `repackage.sh` needs to be adjusted.

It would be an overstatement to describe these as problems, since the links work. The `#readme` anchor advances the scroll down the fold to the description but also allows the reader to scroll up to inspect the files in the root of the repository. With these changes, the files are a click away. I read either as valid intentions.

**Pull request merged**